index.js

```diff
@@ -0,0 +1,77 @@
+import luxon from './modules/luxon.js';
+
+import AwesomeBooks from './modules/book.js';
+
+const awesomeBooks = new AwesomeBooks();
+awesomeBooks.load();
+if (awesomeBooks.getBooks().length < 1) {
+  awesomeBooks.addBook('The Power of Focus', 'Mark Victor Hansen');
+}
+
+const bookListSection = document.querySelector('#book-list');
+
+function renderBookList() {
+  bookListSection.innerHTML = awesomeBooks
+    .getBooks()
+    .map(
+      (book, index) => `
+        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
+            <div>
+                <p class="title">"${book.title}" by ${book.author}</p>
+            </div>
+            <button data-id=${book.id} class="remove">Remove</button>
+        </article>`,
+    )
+    .join('');
+}
+renderBookList();
+
+const addBookForm = document.querySelector('#add-book');
+addBookForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const title = event.target.querySelector('#title').value;
+  const author = event.target.querySelector('#author').value;
+  AwesomeBooks.addBook(title, author);
+  event.target.reset();
```

Comment:
- [ ] Kindly make sure the Add functionality is working properly. There's a small typo here where you typed `AwesomeBooks.addBook(title, author);` but that doesn't work because you're using the class not the object, so it should be `awesomeBooks.addBook(title, author);` which is the object.